content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -215,6 +215,12 @@ aggregationRule:
 rules: [] # Rules are automatically filled in by the controller manager.
 ```
 
+You could also create aggregated ClusterRole via `kubectl`
```

Comment:
would this make more sense in the "command line utilities" section below?

Reply:
Yeah, we have got aggregation clusterrole creation guide in that section.